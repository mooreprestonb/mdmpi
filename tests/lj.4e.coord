<coords>
<natoms> 4 </natoms> <step> 0 </step>
<xyz> 0.999009 0     0           </xyz><vxyz> 10 0 0</vxyz><charge> 1 </charge>
<xyz> 2.99999  0   -9.08203e-06  </xyz><vxyz>0 0 0  </vxyz><charge>-1 </charge>
<xyz> 0.999    0     3           </xyz><vxyz>0 0 0  </vxyz><charge> 1 </charge>
<xyz> 3        0.001 -1.99999    </xyz><vxyz>0 0 0  </vxyz><charge>-1 </charge>
</coords>
