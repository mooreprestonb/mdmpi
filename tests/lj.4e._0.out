Getting initial energy and force
Starting MD ntime = 1000 with 12 dof
<step>        0 
<energyAVG>
<ham>  4.95649e+01 ~  4.95649e+01 </ham>
<ke>  5.00000e+01 ~  5.00000e+01 </ke> 
<pe> -4.35147e-01 ~ -4.35147e-01 </pe>
</energyAVG>
</step>
<step>       10 
<energyAVG>
<ham>  4.95185e+01 ~  4.91299e+01 </ham>
<ke>  5.04902e+01 ~  4.98601e+01 </ke> 
<pe> -9.71623e-01 ~ -7.30223e-01 </pe>
</energyAVG>
</step>
<step>       20 
<energyAVG>
<ham>  4.92502e+01 ~  4.90360e+01 </ham>
<ke>  4.79001e+01 ~  4.99786e+01 </ke> 
<pe>  1.35008e+00 ~ -9.42638e-01 </pe>
</energyAVG>
</step>
<step>       30 
<energyAVG>
<ham>  4.91802e+01 ~  4.90326e+01 </ham>
<ke>  4.84579e+01 ~  4.95014e+01 </ke> 
<pe>  7.22353e-01 ~ -4.68836e-01 </pe>
</energyAVG>
</step>
<step>       40 
<energyAVG>
<ham>  4.91832e+01 ~  4.92323e+01 </ham>
<ke>  4.87035e+01 ~  4.94545e+01 </ke> 
<pe>  4.79687e-01 ~ -2.22193e-01 </pe>
</energyAVG>
</step>
<step>       50 
<energyAVG>
<ham>  4.91928e+01 ~  4.92323e+01 </ham>
<ke>  4.88513e+01 ~  4.94629e+01 </ke> 
<pe>  3.41477e-01 ~ -2.30555e-01 </pe>
</energyAVG>
</step>
<step>       60 
<energyAVG>
<ham>  4.92091e+01 ~  4.92329e+01 </ham>
<ke>  4.89540e+01 ~  4.94947e+01 </ke> 
<pe>  2.55073e-01 ~ -2.61766e-01 </pe>
</energyAVG>
</step>
<step>       70 
<energyAVG>
<ham>  4.92124e+01 ~  4.92329e+01 </ham>
<ke>  4.90322e+01 ~  4.95218e+01 </ke> 
<pe>  1.80266e-01 ~ -2.88867e-01 </pe>
</energyAVG>
</step>
<step>       80 
<energyAVG>
<ham>  4.92150e+01 ~  4.92329e+01 </ham>
<ke>  4.90950e+01 ~  4.95578e+01 </ke> 
<pe>  1.20013e-01 ~ -3.24847e-01 </pe>
</energyAVG>
</step>
<step>       90 
<energyAVG>
<ham>  4.92169e+01 ~  4.92329e+01 </ham>
<ke>  4.91486e+01 ~  4.96068e+01 </ke> 
<pe>  6.83082e-02 ~ -3.73889e-01 </pe>
</energyAVG>
</step>
<step>      100 
<energyAVG>
<ham>  4.92185e+01 ~  4.92329e+01 </ham>
<ke>  4.91976e+01 ~  4.96768e+01 </ke> 
<pe>  2.09357e-02 ~ -4.43842e-01 </pe>
</energyAVG>
</step>
<step>      110 
<energyAVG>
<ham>  4.92198e+01 ~  4.92330e+01 </ham>
<ke>  4.92457e+01 ~  4.97840e+01 </ke> 
<pe> -2.58725e-02 ~ -5.50993e-01 </pe>
</energyAVG>
</step>
<step>      120 
<energyAVG>
<ham>  4.92209e+01 ~  4.92330e+01 </ham>
<ke>  4.92977e+01 ~  4.99668e+01 </ke> 
<pe> -7.68390e-02 ~ -7.33769e-01 </pe>
</energyAVG>
</step>
<step>      130 
<energyAVG>
<ham>  4.92218e+01 ~  4.92332e+01 </ham>
<ke>  4.93623e+01 ~  5.03263e+01 </ke> 
<pe> -1.40418e-01 ~ -1.09310e+00 </pe>
</energyAVG>
</step>
<step>      140 
<energyAVG>
<ham>  4.92226e+01 ~  4.92295e+01 </ham>
<ke>  4.94534e+01 ~  5.08711e+01 </ke> 
<pe> -2.30871e-01 ~ -1.64155e+00 </pe>
</energyAVG>
</step>
<step>      150 
<energyAVG>
<ham>  4.92112e+01 ~  4.90295e+01 </ham>
<ke>  4.94952e+01 ~  4.99515e+01 </ke> 
<pe> -2.83990e-01 ~ -9.21998e-01 </pe>
</energyAVG>
</step>
<step>      160 
<energyAVG>
<ham>  4.92111e+01 ~  4.92331e+01 </ham>
<ke>  4.95705e+01 ~  5.06434e+01 </ke> 
<pe> -3.59424e-01 ~ -1.41034e+00 </pe>
</energyAVG>
</step>
<step>      170 
<energyAVG>
<ham>  4.92124e+01 ~  4.92333e+01 </ham>
<ke>  4.96151e+01 ~  5.01279e+01 </ke> 
<pe> -4.02648e-01 ~ -8.94533e-01 </pe>
</energyAVG>
</step>
<step>      180 
<energyAVG>
<ham>  4.92136e+01 ~  4.92332e+01 </ham>
<ke>  4.96349e+01 ~  4.98751e+01 </ke> 
<pe> -4.21364e-01 ~ -6.41916e-01 </pe>
</energyAVG>
</step>
<step>      190 
<energyAVG>
<ham>  4.92146e+01 ~  4.92332e+01 </ham>
<ke>  4.96433e+01 ~  4.97389e+01 </ke> 
<pe> -4.28670e-01 ~ -5.05688e-01 </pe>
</energyAVG>
</step>
<step>      200 
<energyAVG>
<ham>  4.92155e+01 ~  4.92332e+01 </ham>
<ke>  4.96456e+01 ~  4.96556e+01 </ke> 
<pe> -4.30072e-01 ~ -4.22402e-01 </pe>
</energyAVG>
</step>
<step>      210 
<energyAVG>
<ham>  4.92164e+01 ~  4.92332e+01 </ham>
<ke>  4.96445e+01 ~  4.96001e+01 </ke> 
<pe> -4.28183e-01 ~ -3.66956e-01 </pe>
</energyAVG>
</step>
<step>      220 
<energyAVG>
<ham>  4.92171e+01 ~  4.92332e+01 </ham>
<ke>  4.96415e+01 ~  4.95611e+01 </ke> 
<pe> -4.24394e-01 ~ -3.27988e-01 </pe>
</energyAVG>
</step>
<step>      230 
<energyAVG>
<ham>  4.92178e+01 ~  4.92332e+01 </ham>
<ke>  4.96373e+01 ~  4.95329e+01 </ke> 
<pe> -4.19516e-01 ~ -2.99713e-01 </pe>
</energyAVG>
</step>
<step>      240 
<energyAVG>
<ham>  4.92185e+01 ~  4.92332e+01 </ham>
<ke>  4.96325e+01 ~  4.95121e+01 </ke> 
<pe> -4.14049e-01 ~ -2.78911e-01 </pe>
</energyAVG>
</step>
<step>      250 
<energyAVG>
<ham>  4.92190e+01 ~  4.92332e+01 </ham>
<ke>  4.96274e+01 ~  4.94968e+01 </ke> 
<pe> -4.08315e-01 ~ -2.63659e-01 </pe>
</energyAVG>
</step>
<step>      260 
<energyAVG>
<ham>  4.92196e+01 ~  4.92332e+01 </ham>
<ke>  4.96221e+01 ~  4.94859e+01 </ke> 
<pe> -4.02530e-01 ~ -2.52752e-01 </pe>
</energyAVG>
</step>
<step>      270 
<energyAVG>
<ham>  4.92201e+01 ~  4.92332e+01 </ham>
<ke>  4.96169e+01 ~  4.94786e+01 </ke> 
<pe> -3.96844e-01 ~ -2.45410e-01 </pe>
</energyAVG>
</step>
<step>      280 
<energyAVG>
<ham>  4.92205e+01 ~  4.92332e+01 </ham>
<ke>  4.96119e+01 ~  4.94743e+01 </ke> 
<pe> -3.91363e-01 ~ -2.41122e-01 </pe>
</energyAVG>
</step>
<step>      290 
<energyAVG>
<ham>  4.92210e+01 ~  4.92332e+01 </ham>
<ke>  4.96071e+01 ~  4.94727e+01 </ke> 
<pe> -3.86163e-01 ~ -2.39561e-01 </pe>
</energyAVG>
</step>
<step>      300 
<energyAVG>
<ham>  4.92214e+01 ~  4.92332e+01 </ham>
<ke>  4.96027e+01 ~  4.94737e+01 </ke> 
<pe> -3.81303e-01 ~ -2.40536e-01 </pe>
</energyAVG>
</step>
<step>      310 
<energyAVG>
<ham>  4.92218e+01 ~  4.92332e+01 </ham>
<ke>  4.95986e+01 ~  4.94771e+01 </ke> 
<pe> -3.76831e-01 ~ -2.43966e-01 </pe>
</energyAVG>
</step>
<step>      320 
<energyAVG>
<ham>  4.92221e+01 ~  4.92332e+01 </ham>
<ke>  4.95949e+01 ~  4.94830e+01 </ke> 
<pe> -3.72787e-01 ~ -2.49873e-01 </pe>
</energyAVG>
</step>
<step>      330 
<energyAVG>
<ham>  4.92224e+01 ~  4.92332e+01 </ham>
<ke>  4.95917e+01 ~  4.94915e+01 </ke> 
<pe> -3.69208e-01 ~ -2.58387e-01 </pe>
</energyAVG>
</step>
<step>      340 
<energyAVG>
<ham>  4.92228e+01 ~  4.92332e+01 </ham>
<ke>  4.95889e+01 ~  4.95029e+01 </ke> 
<pe> -3.66135e-01 ~ -2.69765e-01 </pe>
</energyAVG>
</step>
<step>      350 
<energyAVG>
<ham>  4.92231e+01 ~  4.92332e+01 </ham>
<ke>  4.95867e+01 ~  4.95176e+01 </ke> 
<pe> -3.63610e-01 ~ -2.84433e-01 </pe>
</energyAVG>
</step>
<step>      360 
<energyAVG>
<ham>  4.92233e+01 ~  4.92332e+01 </ham>
<ke>  4.95850e+01 ~  4.95362e+01 </ke> 
<pe> -3.61691e-01 ~ -3.03061e-01 </pe>
</energyAVG>
</step>
<step>      370 
<energyAVG>
<ham>  4.92236e+01 ~  4.92332e+01 </ham>
<ke>  4.95841e+01 ~  4.95598e+01 </ke> 
<pe> -3.60448e-01 ~ -3.26694e-01 </pe>
</energyAVG>
</step>
<step>      380 
<energyAVG>
<ham>  4.92239e+01 ~  4.92332e+01 </ham>
<ke>  4.95838e+01 ~  4.95902e+01 </ke> 
<pe> -3.59983e-01 ~ -3.57001e-01 </pe>
</energyAVG>
</step>
<step>      390 
<energyAVG>
<ham>  4.92241e+01 ~  4.92332e+01 </ham>
<ke>  4.95845e+01 ~  4.96299e+01 </ke> 
<pe> -3.60442e-01 ~ -3.96759e-01 </pe>
</energyAVG>
</step>
<step>      400 
<energyAVG>
<ham>  4.92243e+01 ~  4.92332e+01 </ham>
<ke>  4.95864e+01 ~  4.96840e+01 </ke> 
<pe> -3.62053e-01 ~ -4.50876e-01 </pe>
</energyAVG>
</step>
<step>      410 
<energyAVG>
<ham>  4.92245e+01 ~  4.92332e+01 </ham>
<ke>  4.95897e+01 ~  4.97620e+01 </ke> 
<pe> -3.65195e-01 ~ -5.28774e-01 </pe>
</energyAVG>
</step>
<step>      420 
<energyAVG>
<ham>  4.92247e+01 ~  4.92332e+01 </ham>
<ke>  4.95953e+01 ~  4.98838e+01 </ke> 
<pe> -3.70554e-01 ~ -6.50615e-01 </pe>
</energyAVG>
</step>
<step>      430 
<energyAVG>
<ham>  4.92249e+01 ~  4.92333e+01 </ham>
<ke>  4.96045e+01 ~  5.00989e+01 </ke> 
<pe> -3.79535e-01 ~ -8.65638e-01 </pe>
</energyAVG>
</step>
<step>      440 
<energyAVG>
<ham>  4.92251e+01 ~  4.92334e+01 </ham>
<ke>  4.96205e+01 ~  5.05333e+01 </ke> 
<pe> -3.95383e-01 ~ -1.29993e+00 </pe>
</energyAVG>
</step>
<step>      450 
<energyAVG>
<ham>  4.92244e+01 ~  4.90257e+01 </ham>
<ke>  4.96465e+01 ~  5.07179e+01 </ke> 
<pe> -4.22137e-01 ~ -1.69221e+00 </pe>
</energyAVG>
</step>
<step>      460 
<energyAVG>
<ham>  4.92202e+01 ~  4.90258e+01 </ham>
<ke>  4.96558e+01 ~  5.07293e+01 </ke> 
<pe> -4.35572e-01 ~ -1.70352e+00 </pe>
</energyAVG>
</step>
<step>      470 
<energyAVG>
<ham>  4.92162e+01 ~  4.90337e+01 </ham>
<ke>  4.96795e+01 ~  5.05343e+01 </ke> 
<pe> -4.63329e-01 ~ -1.50066e+00 </pe>
</energyAVG>
</step>
<step>      480 
<energyAVG>
<ham>  4.92124e+01 ~  4.90336e+01 </ham>
<ke>  4.96919e+01 ~  5.01068e+01 </ke> 
<pe> -4.79469e-01 ~ -1.07320e+00 </pe>
</energyAVG>
</step>
<step>      490 
<energyAVG>
<ham>  4.92124e+01 ~  4.92332e+01 </ham>
<ke>  4.96977e+01 ~  4.98968e+01 </ke> 
<pe> -4.85321e-01 ~ -6.63534e-01 </pe>
</energyAVG>
</step>
<step>      500 
<energyAVG>
<ham>  4.92128e+01 ~  4.92332e+01 </ham>
<ke>  4.97003e+01 ~  4.97797e+01 </ke> 
<pe> -4.87495e-01 ~ -5.46504e-01 </pe>
</energyAVG>
</step>
<step>      510 
<energyAVG>
<ham>  4.92101e+01 ~  4.90335e+01 </ham>
<ke>  4.97010e+01 ~  4.97061e+01 </ke> 
<pe> -4.90936e-01 ~ -6.72616e-01 </pe>
</energyAVG>
</step>
<step>      520 
<energyAVG>
<ham>  4.92067e+01 ~  4.90335e+01 </ham>
<ke>  4.97006e+01 ~  4.96556e+01 </ke> 
<pe> -4.93863e-01 ~ -6.22179e-01 </pe>
</energyAVG>
</step>
<step>      530 
<energyAVG>
<ham>  4.92034e+01 ~  4.90335e+01 </ham>
<ke>  4.96993e+01 ~  4.96192e+01 </ke> 
<pe> -4.95884e-01 ~ -5.85709e-01 </pe>
</energyAVG>
</step>
<step>      540 
<energyAVG>
<ham>  4.92003e+01 ~  4.90335e+01 </ham>
<ke>  4.96976e+01 ~  4.95918e+01 </ke> 
<pe> -4.97255e-01 ~ -5.58334e-01 </pe>
</energyAVG>
</step>
<step>      550 
<energyAVG>
<ham>  4.91973e+01 ~  4.90335e+01 </ham>
<ke>  4.96954e+01 ~  4.95707e+01 </ke> 
<pe> -4.98145e-01 ~ -5.37282e-01 </pe>
</energyAVG>
</step>
<step>      560 
<energyAVG>
<ham>  4.91944e+01 ~  4.90334e+01 </ham>
<ke>  4.96930e+01 ~  4.95543e+01 </ke> 
<pe> -4.98676e-01 ~ -5.20870e-01 </pe>
</energyAVG>
</step>
<step>      570 
<energyAVG>
<ham>  4.91915e+01 ~  4.90334e+01 </ham>
<ke>  4.96905e+01 ~  4.95415e+01 </ke> 
<pe> -4.98936e-01 ~ -5.08020e-01 </pe>
</energyAVG>
</step>
<step>      580 
<energyAVG>
<ham>  4.91888e+01 ~  4.90334e+01 </ham>
<ke>  4.96878e+01 ~  4.95315e+01 </ke> 
<pe> -4.98995e-01 ~ -4.98023e-01 </pe>
</energyAVG>
</step>
<step>      590 
<energyAVG>
<ham>  4.91862e+01 ~  4.90334e+01 </ham>
<ke>  4.96851e+01 ~  4.95238e+01 </ke> 
<pe> -4.98904e-01 ~ -4.90396e-01 </pe>
</energyAVG>
</step>
<step>      600 
<energyAVG>
<ham>  4.91836e+01 ~  4.90334e+01 </ham>
<ke>  4.96824e+01 ~  4.95183e+01 </ke> 
<pe> -4.98709e-01 ~ -4.84811e-01 </pe>
</energyAVG>
</step>
<step>      610 
<energyAVG>
<ham>  4.91812e+01 ~  4.90334e+01 </ham>
<ke>  4.96796e+01 ~  4.95145e+01 </ke> 
<pe> -4.98445e-01 ~ -4.81048e-01 </pe>
</energyAVG>
</step>
<step>      620 
<energyAVG>
<ham>  4.91788e+01 ~  4.90334e+01 </ham>
<ke>  4.96770e+01 ~  4.95124e+01 </ke> 
<pe> -4.98144e-01 ~ -4.78967e-01 </pe>
</energyAVG>
</step>
<step>      630 
<energyAVG>
<ham>  4.91765e+01 ~  4.90334e+01 </ham>
<ke>  4.96743e+01 ~  4.95119e+01 </ke> 
<pe> -4.97834e-01 ~ -4.78494e-01 </pe>
</energyAVG>
</step>
<step>      640 
<energyAVG>
<ham>  4.91743e+01 ~  4.90334e+01 </ham>
<ke>  4.96718e+01 ~  4.95131e+01 </ke> 
<pe> -4.97540e-01 ~ -4.79613e-01 </pe>
</energyAVG>
</step>
<step>      650 
<energyAVG>
<ham>  4.91721e+01 ~  4.90334e+01 </ham>
<ke>  4.96694e+01 ~  4.95158e+01 </ke> 
<pe> -4.97286e-01 ~ -4.82363e-01 </pe>
</energyAVG>
</step>
<step>      660 
<energyAVG>
<ham>  4.91700e+01 ~  4.90334e+01 </ham>
<ke>  4.96671e+01 ~  4.95203e+01 </ke> 
<pe> -4.97095e-01 ~ -4.86843e-01 </pe>
</energyAVG>
</step>
<step>      670 
<energyAVG>
<ham>  4.91680e+01 ~  4.90334e+01 </ham>
<ke>  4.96650e+01 ~  4.95267e+01 </ke> 
<pe> -4.96992e-01 ~ -4.93223e-01 </pe>
</energyAVG>
</step>
<step>      680 
<energyAVG>
<ham>  4.91660e+01 ~  4.90334e+01 </ham>
<ke>  4.96630e+01 ~  4.95352e+01 </ke> 
<pe> -4.97003e-01 ~ -5.01763e-01 </pe>
</energyAVG>
</step>
<step>      690 
<energyAVG>
<ham>  4.91641e+01 ~  4.90334e+01 </ham>
<ke>  4.96612e+01 ~  4.95463e+01 </ke> 
<pe> -4.97157e-01 ~ -5.12846e-01 </pe>
</energyAVG>
</step>
<step>      700 
<energyAVG>
<ham>  4.91622e+01 ~  4.90334e+01 </ham>
<ke>  4.96597e+01 ~  4.95605e+01 </ke> 
<pe> -4.97488e-01 ~ -5.27036e-01 </pe>
</energyAVG>
</step>
<step>      710 
<energyAVG>
<ham>  4.91604e+01 ~  4.90335e+01 </ham>
<ke>  4.96584e+01 ~  4.95786e+01 </ke> 
<pe> -4.98038e-01 ~ -5.45171e-01 </pe>
</energyAVG>
</step>
<step>      720 
<energyAVG>
<ham>  4.91587e+01 ~  4.90335e+01 </ham>
<ke>  4.96575e+01 ~  4.96020e+01 </ke> 
<pe> -4.98863e-01 ~ -5.68533e-01 </pe>
</energyAVG>
</step>
<step>      730 
<energyAVG>
<ham>  4.91569e+01 ~  4.90335e+01 </ham>
<ke>  4.96570e+01 ~  4.96326e+01 </ke> 
<pe> -5.00037e-01 ~ -5.99170e-01 </pe>
</energyAVG>
</step>
<step>      740 
<energyAVG>
<ham>  4.91553e+01 ~  4.90335e+01 </ham>
<ke>  4.96569e+01 ~  4.96740e+01 </ke> 
<pe> -5.01667e-01 ~ -6.40535e-01 </pe>
</energyAVG>
</step>
<step>      750 
<energyAVG>
<ham>  4.91536e+01 ~  4.90335e+01 </ham>
<ke>  4.96576e+01 ~  4.97324e+01 </ke> 
<pe> -5.03920e-01 ~ -6.98885e-01 </pe>
</energyAVG>
</step>
<step>      760 
<energyAVG>
<ham>  4.91521e+01 ~  4.90335e+01 </ham>
<ke>  4.96591e+01 ~  4.98202e+01 </ke> 
<pe> -5.07074e-01 ~ -7.86689e-01 </pe>
</energyAVG>
</step>
<step>      770 
<energyAVG>
<ham>  4.91505e+01 ~  4.90335e+01 </ham>
<ke>  4.96622e+01 ~  4.99655e+01 </ke> 
<pe> -5.11651e-01 ~ -9.31952e-01 </pe>
</energyAVG>
</step>
<step>      780 
<energyAVG>
<ham>  4.91490e+01 ~  4.90336e+01 </ham>
<ke>  4.96678e+01 ~  5.02400e+01 </ke> 
<pe> -5.18762e-01 ~ -1.20634e+00 </pe>
</energyAVG>
</step>
<step>      790 
<energyAVG>
<ham>  4.91476e+01 ~  4.90330e+01 </ham>
<ke>  4.96785e+01 ~  5.07861e+01 </ke> 
<pe> -5.30942e-01 ~ -1.75305e+00 </pe>
</energyAVG>
</step>
<step>      800 
<energyAVG>
<ham>  4.91461e+01 ~  4.90389e+01 </ham>
<ke>  4.96894e+01 ~  4.96516e+01 </ke> 
<pe> -5.43329e-01 ~ -6.12642e-01 </pe>
</energyAVG>
</step>
<step>      810 
<energyAVG>
<ham>  4.91447e+01 ~  4.90317e+01 </ham>
<ke>  4.96982e+01 ~  5.09015e+01 </ke> 
<pe> -5.53477e-01 ~ -1.86984e+00 </pe>
</energyAVG>
</step>
<step>      820 
<energyAVG>
<ham>  4.91433e+01 ~  4.90337e+01 </ham>
<ke>  4.97089e+01 ~  5.03317e+01 </ke> 
<pe> -5.65607e-01 ~ -1.29804e+00 </pe>
</energyAVG>
</step>
<step>      830 
<energyAVG>
<ham>  4.91420e+01 ~  4.90335e+01 </ham>
<ke>  4.97141e+01 ~  5.00104e+01 </ke> 
<pe> -5.72064e-01 ~ -9.76836e-01 </pe>
</energyAVG>
</step>
<step>      840 
<energyAVG>
<ham>  4.91407e+01 ~  4.90335e+01 </ham>
<ke>  4.97164e+01 ~  4.98455e+01 </ke> 
<pe> -5.75704e-01 ~ -8.12008e-01 </pe>
</energyAVG>
</step>
<step>      850 
<energyAVG>
<ham>  4.91395e+01 ~  4.90335e+01 </ham>
<ke>  4.97173e+01 ~  4.97484e+01 </ke> 
<pe> -5.77809e-01 ~ -7.14915e-01 </pe>
</energyAVG>
</step>
<step>      860 
<energyAVG>
<ham>  4.91382e+01 ~  4.90335e+01 </ham>
<ke>  4.97172e+01 ~  4.96850e+01 </ke> 
<pe> -5.78974e-01 ~ -6.51525e-01 </pe>
</energyAVG>
</step>
<step>      870 
<energyAVG>
<ham>  4.91370e+01 ~  4.90335e+01 </ham>
<ke>  4.97165e+01 ~  4.96406e+01 </ke> 
<pe> -5.79512e-01 ~ -6.07127e-01 </pe>
</energyAVG>
</step>
<step>      880 
<energyAVG>
<ham>  4.91359e+01 ~  4.90335e+01 </ham>
<ke>  4.97155e+01 ~  4.96080e+01 </ke> 
<pe> -5.79613e-01 ~ -5.74510e-01 </pe>
</energyAVG>
</step>
<step>      890 
<energyAVG>
<ham>  4.91347e+01 ~  4.90335e+01 </ham>
<ke>  4.97141e+01 ~  4.95832e+01 </ke> 
<pe> -5.79397e-01 ~ -5.49769e-01 </pe>
</energyAVG>
</step>
<step>      900 
<energyAVG>
<ham>  4.91336e+01 ~  4.90334e+01 </ham>
<ke>  4.97125e+01 ~  4.95641e+01 </ke> 
<pe> -5.78947e-01 ~ -5.30623e-01 </pe>
</energyAVG>
</step>
<step>      910 
<energyAVG>
<ham>  4.91325e+01 ~  4.90334e+01 </ham>
<ke>  4.97108e+01 ~  4.95491e+01 </ke> 
<pe> -5.78323e-01 ~ -5.15655e-01 </pe>
</energyAVG>
</step>
<step>      920 
<energyAVG>
<ham>  4.91314e+01 ~  4.90334e+01 </ham>
<ke>  4.97090e+01 ~  4.95374e+01 </ke> 
<pe> -5.77570e-01 ~ -5.03949e-01 </pe>
</energyAVG>
</step>
<step>      930 
<energyAVG>
<ham>  4.91304e+01 ~  4.90334e+01 </ham>
<ke>  4.97071e+01 ~  4.95283e+01 </ke> 
<pe> -5.76724e-01 ~ -4.94892e-01 </pe>
</energyAVG>
</step>
<step>      940 
<energyAVG>
<ham>  4.91293e+01 ~  4.90334e+01 </ham>
<ke>  4.97051e+01 ~  4.95215e+01 </ke> 
<pe> -5.75812e-01 ~ -4.88066e-01 </pe>
</energyAVG>
</step>
<step>      950 
<energyAVG>
<ham>  4.91283e+01 ~  4.90334e+01 </ham>
<ke>  4.97032e+01 ~  4.95166e+01 </ke> 
<pe> -5.74860e-01 ~ -4.83189e-01 </pe>
</energyAVG>
</step>
<step>      960 
<energyAVG>
<ham>  4.91273e+01 ~  4.90334e+01 </ham>
<ke>  4.97012e+01 ~  4.95135e+01 </ke> 
<pe> -5.73887e-01 ~ -4.80072e-01 </pe>
</energyAVG>
</step>
<step>      970 
<energyAVG>
<ham>  4.91264e+01 ~  4.90334e+01 </ham>
<ke>  4.96993e+01 ~  4.95120e+01 </ke> 
<pe> -5.72911e-01 ~ -4.78602e-01 </pe>
</energyAVG>
</step>
<step>      980 
<energyAVG>
<ham>  4.91254e+01 ~  4.90334e+01 </ham>
<ke>  4.96974e+01 ~  4.95122e+01 </ke> 
<pe> -5.71949e-01 ~ -4.78728e-01 </pe>
</energyAVG>
</step>
<step>      990 
<energyAVG>
<ham>  4.91245e+01 ~  4.90334e+01 </ham>
<ke>  4.96955e+01 ~  4.95139e+01 </ke> 
<pe> -5.71017e-01 ~ -4.80453e-01 </pe>
</energyAVG>
</step>
<step>     1000 
<energyAVG>
<ham>  4.91236e+01 ~  4.90334e+01 </ham>
<ke>  4.96937e+01 ~  4.95173e+01 </ke> 
<pe> -5.70129e-01 ~ -4.83840e-01 </pe>
</energyAVG>
</step>

